sdk/servicebus/azure-messaging-servicebus-track1-perf/src/main/java/com/azure/messaging/servicebus/perf/SendMessagesTest.java

```diff
@@ -0,0 +1,60 @@
+package com.azure.messaging.servicebus.perf;
+
+import com.azure.messaging.servicebus.perf.core.ServiceBusStressOptions;
+import com.azure.messaging.servicebus.perf.core.ServiceTest;
+import com.microsoft.azure.servicebus.IMessage;
+import com.microsoft.azure.servicebus.Message;
+import com.microsoft.azure.servicebus.ReceiveMode;
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+
+public class SendMessagesTest extends ServiceTest<ServiceBusStressOptions> {
+    private List<IMessage> messages = new ArrayList<>();
+
+    public SendMessagesTest(ServiceBusStressOptions options) throws InterruptedException, ExecutionException, ServiceBusException {
+        super(options, ReceiveMode.PEEKLOCK);
+    }
+
+
+
+    public Mono<Void> globalSetupAsync() {
+        String messageId = UUID.randomUUID().toString();
+        Message message = new Message(CONTENTS);
+        message.setMessageId(messageId);
+        return Flux.range(0, options.getMessagesToSend())
```

Comment:
Why are we using a flatmap here? It can easily be an IntStream. `messages` is not thread-safe either, so I would not expect us to use it in a flatMap.